imports/ui/accounts/accounts-templates.html

```diff
@@ -36,7 +36,7 @@
 
 <template name="override-atTitle">
   <h1 class="title-auth">{{title}}</h1>
-  <p class="subtitle-auth">Signing in allows you to have private lists</p>
+  <p class="subtitle-auth">{{_ 'Signing in allows you to have private lists'}}</p>
```

Comment:
I'm not sure what the best practice is here - is it to make this some kind of key, and then have an English translation? The problem here is that if you change the english text you have to change the i18n JSON file as well to get the translation to work for other languages.